packages/runtime/src/core.ts

```diff
@@ -779,22 +805,24 @@ export class FederationHost {
     const sharedKeys = Object.keys(formatShareOptions);
     sharedKeys.forEach((sharedKey) => {
       const sharedVal = formatShareOptions[sharedKey];
-      const registeredShared = getRegisteredShare(
-        this.shareScopeMap,
-        sharedKey,
-        sharedVal,
-        this.hooks.lifecycle.resolveShare,
-      );
-      if (!registeredShared && sharedVal && sharedVal.lib) {
-        this.setShared({
-          pkgName: sharedKey,
-          lib: sharedVal.lib,
-          get: sharedVal.get,
-          loaded: true,
-          shared: sharedVal,
-          from: userOptions.name,
-        });
-      }
+      sharedVal.forEach((s) => {
```

Comment:
Change the variable s to sharedItem. Don’t make it too short, which will cause subsequent functions to become longer and make it difficult to retrieve dependencies.